deployment/CP05MOAS-GL379_Deploy.csv

```diff
@@ -29,9 +29,9 @@ P7a,,,,CP05MOAS-GL379-05-PARADM000,4,1,2017-01-16T12:46:00,2017-01-16T23:59:59,C
 #P7b,,,,CP05MOAS-GL379-03-CTDGVM000,5,1,2017-04-19T20:21:00,2017-04-19T00:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09363,40,-70,,1000,1000,not deployed as planned
 #P7b,,,,CP05MOAS-GL379-04-DOSTAM000,5,1,2017-04-19T20:21:00,2017-04-19T00:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00029,40,-70,,1000,1000,not deployed as planned
 #P7b,,,,CP05MOAS-GL379-05-PARADM000,5,1,2017-04-19T20:21:00,2017-04-19T00:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50238,40,-70,,1000,1000,not deployed as planned
-TS20190205,,,,CP05MOAS-GL379-00-ENG000000,6,1,2019-02-05T07:54:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.9166,-70.36972222,,1000,1000,SS-1
-TS20190205,,,,CP05MOAS-GL379-01-ADCPAM000,6,1,2019-02-05T07:54:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49982,39.9166,-70.36972222,,1000,1000,SS-1
-TS20190205,,,,CP05MOAS-GL379-02-FLORTM000,6,1,2019-02-05T07:54:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-04467,39.9166,-70.36972222,,1000,1000,SS-1
-TS20190205,,,,CP05MOAS-GL379-03-CTDGVM000,6,1,2019-02-05T07:54:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09363,39.9166,-70.36972222,,1000,1000,SS-1
-TS20190205,,,,CP05MOAS-GL379-04-DOSTAM000,6,1,2019-02-05T07:54:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,39.9166,-70.36972222,,1000,1000,SS-1
-TS20190205,,,,CP05MOAS-GL379-05-PARADM000,6,1,2019-02-05T07:54:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50238,39.9166,-70.36972222,,1000,1000,SS-1
\ No newline at end of file
+TS20190205,,TS20190516,,CP05MOAS-GL379-00-ENG000000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.9166,-70.36972222,,1000,1000,SS-1
+TS20190205,,TS20190516,,CP05MOAS-GL379-01-ADCPAM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49982,39.9166,-70.36972222,,1000,1000,SS-1
+TS20190205,,TS20190516,,CP05MOAS-GL379-02-FLORTM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-04467,39.9166,-70.36972222,,1000,1000,SS-1
+TS20190205,,TS20190516,,CP05MOAS-GL379-03-CTDGVM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09363,39.9166,-70.36972222,,1000,1000,SS-1
+TS20190205,,TS20190516,,CP05MOAS-GL379-04-DOSTAM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,39.9166,-70.36972222,,1000,1000,SS-1
+TS20190205,,TS20190516,,CP05MOAS-GL379-05-PARADM000,6,1,2019-02-05T07:54:00,2019-05-16T22:54:03,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50238,39.9166,-70.36972222,,1000,1000,SS-1
```

Comment:
Reviewed by RT - looks good (although QLR has not been posted to Vault yet for documented confirmation of recover date/time).

Reply:
Confirm Cruise UID and recovery time with QLR.